scripts/multiqc

```diff
@@ -181,6 +187,22 @@ make_data_dir, data_format, zip_data_dir, force, plots_flat, plots_interactive,
     config.plots_force_flat = plots_flat
     config.plots_force_interactive = plots_interactive
     config.kwargs = kwargs # Plugin command line options
+
+    # Add files if --is-file-list option is given
+    if is_file_list:
+        if len(analysis_dir) > 1:
+            raise ValueError("If --is-file-list is giving, analysis_dir should have only one plain text file.")
+        if os.path.isdir(analysis_dir[0]):
+            raise ValueError("If --is-file-list is giving, analysis_dir should be a plain text file.")
+        config.analysis_dir = ()
+        with (open(analysis_dir[0])) as in_handle:
+            for line in in_handle:
+                if os.path.exists(line.strip()):
+                    config.analysis_dir += (os.path.abspath(line.strip()),)
+        if len(config.analysis_dir) == 0:
+            logger.warning("No files were added from %s using --is-file-list option." % analysis_dir[0])
```

Comment:
Small thing, but I prefer using `format` syntax as in the rest of the program:
```python
logger.error("No files were added from {} using --is-file-list option.".format(analysis_dir[0]))
```
Also, this should probably be a `Raise` as the program can't continue without any files.